skillserver/echo.go

```diff
@@ -24,18 +26,22 @@ func (this *EchoRequest) VerifyAppID(myAppID string) bool {
 	return false
 }
 
+// Deprecated: Please use the github.com/mikeflynn/go-alexa/customskill package
 func (this *EchoRequest) GetSessionID() string {
 	return this.Session.SessionID
 }
 
+// Deprecated: Please use the github.com/mikeflynn/go-alexa/customskill package
 func (this *EchoRequest) GetUserID() string {
```

Comment:
I didn't add this to `customskill`. I'm not convinced we need these little helper functions.